content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -763,4 +770,8 @@ Delaying volume binding allows the scheduler to consider all of a pod's
 scheduling constraints when choosing an appropriate PersistentVolume for a
 PersistentVolumeClaim.
 
+### Note
+
+You should only have one default storage class in your cluster at any given time. Kubernetes does not technically prevent you from having more than one storage class; however, requests for persistent volume claims will fail when there are multiple default storage classes.
```

Comment:
How about something like:
```
There can only be one default storage class in the cluster.  Dynamic provisioning will fail if there are multiple default storage classes set.
```

Also move this under the "default storage class" section.